docs/es/cookbook/components/index.md

```diff
@@ -1,9 +1,9 @@
-# Componentes
+# Components
```

Comment:
Why is it translated back to english ?

Reply:
merge shenanigans 🙃 if you found more of this let me know to change it